setup.py

```diff
@@ -14,15 +14,16 @@
     author_email='info@exabyte.io',
     packages=find_packages(exclude=["tests.*", "tests"]),
     install_requires=[
-        "bunch==1.0.1",
+        "munch==2.5.0",
         "numpy==1.17.3",
         "pymatgen==2020.4.29",
         "ase==3.17.0",
         "esse>=2020.10.19",
     ],
     extras_require={
         "test": [
-            "mock==1.3.0",
+            "mock>=1.3.0",
+            "pyyaml>=4.2b1,<6",
```

Comment:
(1) Below, the `references.py`, `__init__.py` x 2, should be included as sources instead of LFS.
(2) Do we really need `run.sh` below?

Reply:
1) I made the corrections to the git lfs files
2) As for the run.sh file, based on our conversation i don't think it's needed. I was modeling the nwchem dir after the vasp and espresso dirs which each have the file present.

Reply:
Fixed the git lfs issue

we have run.sh in both espresso and vasp on the dev branch. If they are not supposed to be there I can fix that. Per our discussion I don't think they are necessary. I just built the nwchem setup based on the espresso/vasp set up.